apps/src/weblab2/Config.tsx

```diff
@@ -1,157 +0,0 @@
-import {ConfigType} from '@codebridge/types';
-import React, {useState} from 'react';
-
-import './styles/config.css';
-import {MultiFileSource} from '@cdo/apps/lab2/types';
-
-type BaseConfigProps = {configName: string; cancelConfig: () => void};
-
-type ConfigProps = BaseConfigProps & {
```

Comment:
We used to have it be possible to provide a grid layout live to web lab 2. This didn't make sense long term, and we also no longer support grid layouts (so we can have adjustable panels)